nix/offchain.nix

```
@@ -232,6 +232,14 @@ in
                 type = types.path;
               };
 
+              strictComp = lib.mkOption {
+                description = ''
+                  Whether to compile the PureScript source with `--strict`    
```

Could you add `Added in: 2.8.0`?

```suggestion
                  Whether to compile the PureScript source with `--strict`
                  
                  Added in: 2.8.0.
```